telegram/ext/dispatcher.py

```diff
@@ -90,11 +90,16 @@ class Dispatcher(object):
             handlers
         update_queue (Queue): The synchronized queue that will contain the
             updates.
+        job_queue (Optional[telegram.ext.JobQueue]): The ``JobQueue`` instance to pass onto handler
+            callbacks
+        workers (Optional[int]): Number of maximum concurrent worker threads for the ``@run_async``
+            decorator
     """
 
-    def __init__(self, bot, update_queue, workers=4, exception_event=None):
+    def __init__(self, bot, update_queue, job_queue=None, workers=4, exception_event=None):
```

Comment:
@jh0ker
unless there's a good reason to break backward compatibility here I think it's better be the last keyword arg